### ISSUE TITLE
Modul funktioniert nicht wenn changelanguage installiert ist?

### ISSUE THREAD

**Issue by Kahmoon** [null0815/contao-summary#1]
Hi,

ich habe bei einer Installation changelanguage nachträglich installiert. Sobald changelanguage aktiv ist funktioniert diese Erweiterung nicht mehr. Irgendwas kommt sich da ins Gehege und ist reproduzierbar.

VG
Frank

**Comment by Kahmoon** [comment]
**Nachtrag** 
Die Zusatzinformationen verschwinden nur bei der Seitenstruktur und Artikeln. Bei einzelnen Inhaltselementen werden sie noch angezeigt.

**Comment by johannes-bittner** [comment]
Kann ich so bestätigen, schließe mich der Bitte um einen Fix an, in der Tat ein Traum, endlich in der Listenansicht schon viel mehr relevante Informationen angezeigt zu bekommen!

**Comment by manu09** [comment]
Hallo, das Problem ist bekannt, da Contao leider nur ein `label_callback` bzw. `child_record_callback` erlaubt (siehe auch contao/core#6235) passiert es schnell, dass die eine Erweiterung die andere überschreibt. Werde an einer Lösung arbeiten, die changelanguage miteinbezieht.

**Comment by Kahmoon** [comment]
Top. Danke! :-)

Würde es helfen deine Erweiterung mit **x_deinModulname** zu verwenden damit es erst ganz am Ende geladen wird und CL somit überschreibt?

**Comment by manu09** [comment]
Nein, soweit ich weiß nicht. changelanguage verwendet den [loadDataContainer](https://github.com/terminal42/contao-changelanguage/blob/ad4269ece6918109cc4361bb4bf46a7fa9441ed7/library/Terminal42/ChangeLanguage/EventListener/CallbackSetupListener.php#L51) Hook. Von daher wäre der Modulname ohnehin irrelevant.

**Comment by Kahmoon** [comment]
Schade :-). Ist mir gerade eingefallen das man da so teilweise austricksen kann.
